src/coreclr/vm/class.cpp

```diff
@@ -2235,8 +2235,8 @@ void MethodTable::DebugRecursivelyDumpInstanceFields(LPCUTF8 pszClassName, BOOL
         {
             // Display them
             if(debug) {
-                ssBuff.Printf(W("%S:\n"), pszClassName);
-                WszOutputDebugString(ssBuff.GetUnicode());
+                ssBuff.Printf("%s:\n", pszClassName);
+                OutputDebugStringUtf8(ssBuff.GetUTF8NoConvert());
```

Comment:
It may be nice to add `GetUTF8()` to SString that does the conversion as necessary. Same as the current `GetUnicode()` method.

Reply:
The `GetUTF8NoConvert()` is basically just that. It throws an exception if done incorrectly though. Fixing some of these API issues is what @jkoritzinsky is mulling over.